.github/workflows/cd_support_sphere_py.yml

```diff
@@ -93,3 +95,39 @@ jobs:
 
       - uses: pypa/gh-action-pypi-publish@v1.9.0
         if: startsWith(github.ref, 'refs/tags')
+
+  push_to_registry:
+    name: Push Docker image to Docker Hub
```

Comment:
nit: We're pushing to ghcr, not Docker Hub, so might be worth changing the step name here to account for that -- otherwise could be confusing

Reply:
Updated. Thanks! :)